src/app/admin/user-panel/user-create/create-user.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="create-user-dialog">
+  <h2 matDialogTitle>Create a new User</h2>
+
+  <form [formGroup]="userForm" (ngSubmit)="onSubmit()">
+    <mat-dialog-content>
+      <div class="form-fields">
+        <mat-form-field appearance="outline">
+          <mat-label>Email Address</mat-label>
+          <input
+            matInput
+            formControlName="email"
+            type="email"
+            autocomplete="email"
```

Comment:
I wonder if we should have autocomplete off in this admin context.

Reply:
yeah... good point